app/models/edm/provided_cho.rb

```diff
@@ -52,6 +53,7 @@ class ProvidedCHO
     has_rdf_predicate :dc_subject_agents, RDF::Vocab::DC11.subject
 
     graphs_without RDF::Vocab::EDM.wasPresentAt
+    dereferences RDF::Vocab::DC.spatial, only: %r(\Ahttp://data.europeana.eu/place/base)
```

Comment:
Could we loosen this criterion to be `%r(\Ahttp://data.europeana.eu/place/)`, because it is intended later this year to remove "/base" from entity URIs.